src/commands/admin.rs

```diff
@@ -133,3 +133,19 @@ pub fn update(ctx: &mut Context, msg: &Message) -> CommandResult {
         .exec())?;
     Ok(())
 }
+
+#[command]
+pub fn say(ctx: &mut Context, _msg: &Message, mut args: Args) -> CommandResult {
+    let channel_id = args.single::<ChannelId>()?;
+    channel_id.say(&ctx.http, args.rest())?;
+    Ok(())
+}
+
+#[command]
+pub fn edit(ctx: &mut Context, _msg: &Message, mut args: Args) -> CommandResult {
+    let channel_id = args.single::<ChannelId>()?;
+    let msg_id = args.single::<u64>()?;
```

Comment:
Maybe parse to a `MessageId`?
The assembly is the same but it's nicer :stuck_out_tongue:

Reply:
I tried it and it isn't implemented